internal/app/tfsec/rules/azure/appservice/http_logs_enabled_rule.go

```diff
@@ -64,8 +66,12 @@ resource "azurerm_app_service" "good_example" {
 			if logProps.MissingChild("http_logs") {
 				set.AddResult().
 					WithDescription("Resource '%s' does not have logs.http_logs enabled", resourceBlock.FullName()).WithBlock(logProps)
+				return
+			}
+			if logProps.MissingNestedChild("http_logs.file_system") {
```

Comment:
Should this be a check for both types of http_logs? I guess the description would need changing too if you agree
```suggestion
			if logProps.MissingNestedChild("http_logs.file_system") && logProps.MissingNestedChild("http_logs.azure_blob_storage") {
```